priv/repo/migrations/20210819081432_user_like.exs

```diff
@@ -0,0 +1,13 @@
+defmodule BgsiteOfficial.Repo.Migrations.UserLike do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_like) do
+      add(:user_id, references(:user))
+      add(:website_id, references(:website))
+      timestamps()
+    end
+    create unique_index(:user,[:user_id], name: :user_unique_index)
```

Comment:
IMO we will need uniqueness on user_id && website_id, not on the separate columns. Refer to the implementation of `website_tag` table, please.